tools/topology/sof-icl-rt711-rt1308-rt715-hdmi.m4

```diff
@@ -78,24 +79,24 @@ dnl     period, priority, core,
 dnl     pcm_min_rate, pcm_max_rate, pipeline_rate,
 dnl     time_domain, sched_comp)
 
-# Low Latency playback pipeline 6 on PCM 5 using max 2 channels of s32le.
+# Low Latency playback pipeline 6 on PCM 5 using max 2 channels of s24le.
 # Schedule 48 frames per 1000us deadline on core 0 with priority 0
 PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
-	6, 5, 2, s32le,
+	6, 5, 2, s24le,
```

Comment:
There is a known hw requirement for 24bit stream.  In HDA,  24bit sample is transferred in 32bit container. 24bit sample data should be left aligned or MSB aligned. But our hw sends data right aligned  or LSB aligned, @singalsu gave a simple solution that we set dai format to 32le and let volume component to do the conversion, please check sof-hda-generic.m4

And there is another solution: use 16bit pipeline for HDMI. It is used by cAVS topology,

Reply:
Thanks @RanderWang updating now.